src/lib/builders/context-menu/create.ts

```diff
@@ -138,8 +138,8 @@ export function createContextMenu(props?: CreateContextMenuProps) {
 				style: styleToString({
 					display: $isVisible ? undefined : 'none',
 				}),
-				id: ids.menu,
-				'aria-labelledby': ids.trigger,
+				id: get(ids.menu),
```

Comment:
These should be reactive, meaning you should reference ids.menu in `stores`, then use it in `returned`.